ui.content/src/main/content/jcr_root/_oak_index/.content.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:oak="http://jackrabbit.apache.org/oak/ns/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:rep="internal"
+    jcr:mixinTypes="[rep:AccessControllable]"
+    jcr:primaryType="nt:unstructured">
+    <hclecommLuceneIndex
+        jcr:primaryType="oak:QueryIndexDefinition"
+        async="async"
+        compatVersion="{Long}2"
+        evaluatePathRestrictions="{Boolean}true"
+        excludedPaths="[/var,/etc/replication,/etc/workflow/instances,/jcr:system]"
+        fulltextEnabled="{Boolean}true"
+        includedPaths="/content/hclecomm"
+        indexPath="/oak:index/hclecommLuceneIndex"
+        name="hclecommLuceneIndex"
+        queryPaths="/content/hclecomm"
+        reindex="{Boolean}false"
+        reindexCount="{Long}5"
+        seed="{Long}1636749256649033006"
+        type="lucene">
+        <aggregates jcr:primaryType="nt:unstructured">
+            <cq:Page jcr:primaryType="nt:unstructured">
+                <include0
+                    jcr:primaryType="nt:unstructured"
+                    path="jcr:content"
+                    relativeNode="{Boolean}true"/>
+            </cq:Page>
+            <nt:file jcr:primaryType="nt:unstructured">
+                <include0
+                    jcr:primaryType="nt:unstructured"
+                    path="jcr:content"/>
+            </nt:file>
+            <cq:PageContent jcr:primaryType="nt:unstructured">
+                <include0
+                    jcr:primaryType="nt:unstructured"
+                    path="*"/>
+                <include1
+                    jcr:primaryType="nt:unstructured"
+                    path="*/*"/>
+                <include2
+                    jcr:primaryType="nt:unstructured"
+                    path="*/*/*"/>
+                <include3
+                    jcr:primaryType="nt:unstructured"
+                    path="*/*/*/*"/>
+                <include4
+                    jcr:primaryType="nt:unstructured"
+                    path="*/*/*/*/*"/>
+                <include5
+                    jcr:primaryType="nt:unstructured"
+                    path="*/*/*/*/*/*"/>
+                <include6
+                    jcr:primaryType="nt:unstructured"
+                    path="*/*/*/*/*/*/*"/>
+                <include7
+                    jcr:primaryType="nt:unstructured"
+                    path="*/*/*/*/*/*/*/*"/>
+            </cq:PageContent>
+        </aggregates>
+        <indexRules jcr:primaryType="nt:unstructured">
+            <nt:base
```

Comment:
@rakeshkumar-verma : should this not be cq:Page ??

Reply:
@Rajamouli-hcl  updated to cq:page